src/main/java/org/terasology/launcher/packages/PackageManager.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2019 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.terasology.launcher.packages;
+
+import java.nio.file.Path;
+
+/**
+ * Handles installation, removal and update of game packages.
```

Comment:
If this is the central-most class for the concept you might want to beef up the javadoc a fair amount 👍

Reply:
It'll be fully documented by the end :+1: